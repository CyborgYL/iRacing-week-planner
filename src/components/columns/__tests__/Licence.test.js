import React from 'react';
import { shallow } from 'enzyme';
import { describe, test } from '@jest/globals';
import { Licence } from '../index';

describe('components/columns/Licence', () => {
  test('renders correctly', () => {
    const component = shallow(<Licence race={{ licenceLevel: 7 }} />);

    expect(component).toMatchSnapshot();
  });
});
