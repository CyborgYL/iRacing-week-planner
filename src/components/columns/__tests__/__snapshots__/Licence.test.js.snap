// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/columns/Licence renders correctly 1`] = `
<td>
  <LicenceLevel
    effective={false}
    licence={7}
  />
</td>
`;
